docs/stable/14-mentions.md

```diff
@@ -0,0 +1,27 @@
+---
+permalink: /specs/14
+parent: Stable specs
+title: 14/MENTIONS
+---
+
+# 14/MENTIONS
+
+## Usage
+
+To mention a user, one would type a message containing `@0xpk` where `pk` is the public key of the [user account](https://specs.status.im/spec/2) to be mentioned. The message is sent as a [regular message with `TEXT_PLAIN` content type](https://specs.status.im/spec/6#content-types).
```

Comment:
Would the user manually enter the `pk` manually or does the application enter the `pk` in place of a pseudonym?